### DISCUSSION TITLE
Stale bot

### DISCUSSION THREAD

**Discussion by ChipWolf** [Koenkk/zigbee2mqtt#15760] [General]
Any chance we can get rid of stale bot in this repo, it's closing perfectly valid issues
I don't think closing an issue without recent activity aids collaboration in any way

**Comment by Koenkk** [comment]
>  it's closing perfectly valid issues

Some are and (I think) most aren't. I think it is good to keep it, otherwise the issue tracker will quickly have 1000+ issues in it with most being inactive. I can always add the `dont-stale` flag if it gets annoying on a particular issue.
